tools-local/tasks/mobile.tasks/AppleAppBuilder/AppleAppBuilder.cs

```diff
@@ -12,6 +12,12 @@
 
 public class AppleAppBuilderTask : Task
 {
+    /// <summary>
+    /// The Apple OS we are targeting (iOS or tvOS)
+    /// </summary>
+    [Required]
+    public string TargetOS { get; set; } = Utils.TargetOS.iOS;
```

Comment:
Do msbuild task support enums?

Reply:
I don't think so.  Felt that all string comparison was the easiest way to go.